portfolio/src/main/webapp/index.html

```diff
@@ -4,15 +4,42 @@
     <meta charset="UTF-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div id="headerContent">
+      <div id="imageContainer">
+	<img src="images/mainSelfie.jpg"  alt="Picture of me" id="headerSelfie">
+      	<button  id="cycleButton" onclick="cyclePictures()"> Cycle Pictures </button>
+      </div>	
+      <div id="greetingContainer">
+      	<h1>Welcom to my portfolio</h1>
+      	<h3>My name is Malachi Brewer. I am a sophomore at the University of Georgia.</h3>
+	<h3>Welcome Buds! I created this site to house random pieces of information about me for anyone who might 
+           be curious. Clicking on the buttons below will take you to different parts of the page.</h3> 
```

Comment:
This line split seems a bit odd after "might" and before "be", is this intended or some auto-formatter funkiness?

Reply:
I intended to do that so line 17 wouldn't be too long. I changed it so the line splits after "curious." since it's the end of the sentence.